packages/serverless-api/src/client.ts

```diff
@@ -87,7 +88,7 @@ export function createGotClient(config: ClientConfig): GotClient {
     username: username,
     password: password,
     headers: {
-      'User-Agent': 'twilio-serverless-api',
+      'User-Agent': getUserAgent(),
```

Comment:
I think it would be better to pass `config.userAgentExtensions` to `getUserAgent` here, rather than using global scope to store a list of them. It would make `getUserAgent` easier to test and reason about (though I'm not going to call it "pure" given it reaches out to the OS for details).